### ISSUE TITLE
Connection immediatly dropped with 3.0.0

### ISSUE THREAD

**Issue by JasperVanEsveld** [lavaclient/lavadeno#6]
Lavalink version: 3.3.2.5
The connection gets immediately dropped by lavalink for unknown reason.

Events received, logged using same log methods as in testing files:
```
[music] connection: creating websocket...
[music] reconnected to node.
[music] connection: connected in 7ms
[bot] disconnected, code=1011, reason=unknown
[music] rest: GET /routeplanner/status | total requests=1
{ class: null, details: null }
````

The log in lavalink upon connection:
```
2021-08-09 01:57:26.354  INFO 13772 --- [  XNIO-1 task-1] io.undertow.servlet                      : Initializing Spring DispatcherServlet 'dispatcherServlet'
2021-08-09 01:57:26.354  INFO 13772 --- [  XNIO-1 task-1] o.s.web.servlet.DispatcherServlet        : Initializing Servlet 'dispatcherServlet'
2021-08-09 01:57:26.358  INFO 13772 --- [  XNIO-1 task-1] o.s.web.servlet.DispatcherServlet        : Completed initialization in 4 ms
2021-08-09 01:57:26.372  INFO 13772 --- [  XNIO-1 task-1] l.server.io.HandshakeInterceptorImpl     : Incoming connection from /0:0:0:0:0:0:0:1:55500
2021-08-09 01:57:26.415 ERROR 13772 --- [  XNIO-1 task-1] w.s.h.ExceptionWebSocketHandlerDecorator : Closing session due to exception for StandardWebSocketSession[id=d55f36b1-28f2-f373-2682-850468b4dc7e, uri=/]

kotlin.KotlinNullPointerException: null
        at lavalink.server.io.SocketServer.afterConnectionEstablished(SocketServer.kt:72) ~[classes!/:na]
        at org.springframework.web.socket.handler.WebSocketHandlerDecorator.afterConnectionEstablished(WebSocketHandlerDecorator.java:70) ~[spring-websocket-5.1.9.RELEASE.jar!/:5.1.9.RELEASE]
        at org.springframework.web.socket.handler.LoggingWebSocketHandlerDecorator.afterConnectionEstablished(LoggingWebSocketHandlerDecorator.java:48) ~[spring-websocket-5.1.9.RELEASE.jar!/:5.1.9.RELEASE]
        at org.springframework.web.socket.handler.ExceptionWebSocketHandlerDecorator.afterConnectionEstablished(ExceptionWebSocketHandlerDecorator.java:48) ~[spring-websocket-5.1.9.RELEASE.jar!/:5.1.9.RELEASE]
        at org.springframework.web.socket.adapter.standard.StandardWebSocketHandlerAdapter.onOpen(StandardWebSocketHandlerAdapter.java:103) ~[spring-websocket-5.1.9.RELEASE.jar!/:5.1.9.RELEASE]
        at io.undertow.websockets.jsr.EndpointSessionHandler.onConnect(EndpointSessionHandler.java:129) ~[undertow-websockets-jsr-2.0.26.Final.jar!/:2.0.26.Final]
        at io.undertow.websockets.jsr.ServerWebSocketContainer$4.handleUpgrade(ServerWebSocketContainer.java:466) ~[undertow-websockets-jsr-2.0.26.Final.jar!/:2.0.26.Final]
        at io.undertow.server.protocol.http.HttpReadListener.exchangeComplete(HttpReadListener.java:385) ~[undertow-core-2.0.26.Final.jar!/:2.0.26.Final]
        at io.undertow.server.protocol.http.HttpServerConnection.exchangeComplete(HttpServerConnection.java:232) ~[undertow-core-2.0.26.Final.jar!/:2.0.26.Final]
        at io.undertow.server.HttpServerExchange.invokeExchangeCompleteListeners(HttpServerExchange.java:1279) ~[undertow-core-2.0.26.Final.jar!/:2.0.26.Final]
        at io.undertow.server.HttpServerExchange.terminateResponse(HttpServerExchange.java:1563) ~[undertow-core-2.0.26.Final.jar!/:2.0.26.Final]
        at io.undertow.server.Connectors.terminateResponse(Connectors.java:159) ~[undertow-core-2.0.26.Final.jar!/:2.0.26.Final]
        at io.undertow.server.protocol.http.HttpTransferEncoding$3.handleEvent(HttpTransferEncoding.java:197) ~[undertow-core-2.0.26.Final.jar!/:2.0.26.Final]
        at io.undertow.server.protocol.http.HttpTransferEncoding$3.handleEvent(HttpTransferEncoding.java:195) ~[undertow-core-2.0.26.Final.jar!/:2.0.26.Final]
        at io.undertow.conduits.HeadStreamSinkConduit.exitFlush(HeadStreamSinkConduit.java:192) ~[undertow-core-2.0.26.Final.jar!/:2.0.26.Final]
        at io.undertow.conduits.HeadStreamSinkConduit.flush(HeadStreamSinkConduit.java:133) ~[undertow-core-2.0.26.Final.jar!/:2.0.26.Final]
        at org.xnio.conduits.ConduitStreamSinkChannel.flush(ConduitStreamSinkChannel.java:162) ~[xnio-api-3.3.8.Final.jar!/:3.3.8.Final]
        at io.undertow.channels.DetachableStreamSinkChannel.flush(DetachableStreamSinkChannel.java:119) ~[undertow-core-2.0.26.Final.jar!/:2.0.26.Final]
        at org.xnio.channels.Channels.flushBlocking(Channels.java:63) ~[xnio-api-3.3.8.Final.jar!/:3.3.8.Final]
        at io.undertow.servlet.spec.ServletOutputStreamImpl.close(ServletOutputStreamImpl.java:618) ~[undertow-servlet-2.0.26.Final.jar!/:2.0.26.Final]
        at io.undertow.servlet.spec.HttpServletResponseImpl.closeStreamAndWriter(HttpServletResponseImpl.java:528) ~[undertow-servlet-2.0.26.Final.jar!/:2.0.26.Final]
        at io.undertow.servlet.spec.HttpServletResponseImpl.responseDone(HttpServletResponseImpl.java:617) ~[undertow-servlet-2.0.26.Final.jar!/:2.0.26.Final]
        at io.undertow.servlet.handlers.ServletInitialHandler.handleFirstRequest(ServletInitialHandler.java:328) ~[undertow-servlet-2.0.26.Final.jar!/:2.0.26.Final]
        at io.undertow.servlet.handlers.ServletInitialHandler.access$100(ServletInitialHandler.java:78) ~[undertow-servlet-2.0.26.Final.jar!/:2.0.26.Final]
        at io.undertow.servlet.handlers.ServletInitialHandler$2.call(ServletInitialHandler.java:133) ~[undertow-servlet-2.0.26.Final.jar!/:2.0.26.Final]
        at io.undertow.servlet.handlers.ServletInitialHandler$2.call(ServletInitialHandler.java:130) ~[undertow-servlet-2.0.26.Final.jar!/:2.0.26.Final]
        at io.undertow.servlet.core.ServletRequestContextThreadSetupAction$1.call(ServletRequestContextThreadSetupAction.java:48) ~[undertow-servlet-2.0.26.Final.jar!/:2.0.26.Final]       
        at io.undertow.servlet.core.ContextClassLoaderSetupAction$1.call(ContextClassLoaderSetupAction.java:43) ~[undertow-servlet-2.0.26.Final.jar!/:2.0.26.Final]
        at io.undertow.servlet.handlers.ServletInitialHandler.dispatchRequest(ServletInitialHandler.java:249) ~[undertow-servlet-2.0.26.Final.jar!/:2.0.26.Final]
        at io.undertow.servlet.handlers.ServletInitialHandler.access$000(ServletInitialHandler.java:78) ~[undertow-servlet-2.0.26.Final.jar!/:2.0.26.Final]
        at io.undertow.servlet.handlers.ServletInitialHandler$1.handleRequest(ServletInitialHandler.java:99) ~[undertow-servlet-2.0.26.Final.jar!/:2.0.26.Final]
        at io.undertow.server.Connectors.executeRootHandler(Connectors.java:376) ~[undertow-core-2.0.26.Final.jar!/:2.0.26.Final]
        at io.undertow.server.HttpServerExchange$1.run(HttpServerExchange.java:830) ~[undertow-core-2.0.26.Final.jar!/:2.0.26.Final]
        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) ~[na:na]
        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) ~[na:na]
        at java.base/java.lang.Thread.run(Thread.java:830) ~[na:na]
```

**Comment by viztea** [comment]
👌🏼 Thanks for creating an issue, I'm working on a fix.

**Comment by viztea** [comment]
Fixed in 3.0.1
